### PR TITLE
Prettier format

### PR DESCRIPTION
**Pull request by abstractalgo**
Just ran `prettier --write` for all folders in the repo.

### CODE REVIEW

File: test/examples/node/objects.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
-
 function foo() {
```
Comment (jasonLaster) [inline test/examples/node/objects.js:1]: we should not be changing files in `test/examples`

File: src/devtools/client/debugger/src/workers/parser/tests/fixtures/async.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 async function foo() {
```
Comment (jasonLaster) [inline src/devtools/client/debugger/src/workers/parser/tests/fixtures/async.js:1]: we should not be changing files in `src/devtools/client/debugger/src/workers/parser/tests/fixtures`

**Review by jasonLaster — COMMENTED** [review]
Two spots we should not be touching, but otherwise good

**Comment by jasonLaster** [comment]
also some random tsc issues

```
i/utils/intercom.ts(26,24): error TS2339: Property 'intercomSettings' does not exist on type 'Window & typeof globalThis'.
src/ui/utils/intercom.ts(30,13): error TS2339: Property 'c' does not exist on type '() => void'.
src/ui/utils/intercom.ts(32,11): error TS2339: Property 'q' does not exist on type '() => void'.
src/ui/utils/intercom.ts(33,11): error TS2339: Property 'c' does not exist on type '() => void'.
src/ui/utils/intercom.ts(33,25): error TS7006: Parameter 'args' implicitly has an 'any' type.
src/ui/utils/intercom.ts(34,13): error TS2339: Property 'q' does not exist on type '() => void'.
src/ui/utils/intercom.ts(43,11): error TS2531: Object is possibly 'null'.
src/ui/utils/intercom.ts(45,15): error TS2339: Property 'attachEvent' does not exist on type 'Window & typeof globalThis'.
src/ui/utils/intercom.ts(46,13): error TS2339: Property 'attachEvent' does not exist on type 'Window & typeof globalThis'.
```

**Comment by abstractalgo** [comment]
Closing in favor of #5133 .

**Pull request closed without merging**